notebooks/en/issues_in_text_dataset.ipynb

```diff
@@ -0,0 +1,3635 @@
+{
```

Comment:
<p>Let's remove these outputs as they take a lot of space and are not super informative. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>